src/style.css

```diff
@@ -1,3 +1,85 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:500&display=swap';
+
+:root {
+  --blue-color: #00acdc;
+}
+
+html {
+  font-size: 12px;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Consider adding fallback fonts to enhance cross-browser compatibility. For example, if 'Roboto' fails to load, the browser will use 'Arial' or a generic sans-serif font. Example: `font-family: 'Roboto', 'Arial', sans-serif;`.